deepchecks/tabular/context.py

```diff
@@ -213,8 +213,13 @@ def __init__(
             model_type_validation(model)
         if feature_importance is not None and not isinstance(feature_importance, pd.Series):
             raise DeepchecksValueError('feature_importance must be a pandas Series')
+        if model_classes and len(model_classes) == 0:
+            raise DeepchecksValueError('Received empty model_classes')
+        if model_classes and sorted(model_classes) != model_classes:
+            raise DeepchecksValueError('Received unsorted model_classes')
```

Comment:
great validation, add link to supported model (otherwise they will not understand why it must be sorted)